gulpfile.js

```diff
@@ -0,0 +1,56 @@
+const del = require('del');
+const gulp = require('gulp');
+const babel = require('gulp-babel');
+const sourcemaps = require('gulp-sourcemaps');
+const babelConfig = require('./package.json').babel;
+
+function _build({ destination, moduleType }) {
+  return gulp
+    .src('src/**/*.js', { ignore: 'src/**/*.spec.js' })
+    .pipe(sourcemaps.init())
+    .pipe(
+      babel({
+        presets: babelConfig.presets.map((presetConfig) => {
+          const hasOptions = Array.isArray(presetConfig);
+          const presetName = hasOptions ? presetConfig[0] : presetConfig;
+          const presetOptions = hasOptions ? presetConfig[1] : undefined;
+
+          if (presetName !== 'env') {
+            return presetConfig;
+          }
+
+          return [
+            presetName,
+            {
+              ...presetOptions,
```

Comment:
I'm assuming it's fine but will using the spread operator on undefined cause any issues?

Reply:
This works fine, I _checked_